codes/classical/q-ary_digits/rank_modulation.yml

```diff
@@ -0,0 +1,43 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+# code id, physical, logical are all lower case
+code_id: rank_modulation
+physical: groups
+logical: q-ary_digits
+
+
+name: 'Rank modulation code'
+introduced: '\cite{doi:10.1109/ISIT.2008.4595285}'
+
+
+description: |
+    Rank modulation codes are a family of codes which encode an alphabet of size \(q\) into a set \(S_n\) of permutations of \([n]=(1,2,...,n)\).
+    They can be derived from Lee-metric codes, Reed-Solomon codes, quadratic residue codes and most binary codes.
+
+protection: |
+    Protects against \textit{Kendall} errors, which are operators \(e\) on the Kendall metric space \(\mathcal(X_n)=(S_n, d_\tau)\) satisfying \(d_\tau(\sigma, e(\sigma))=1,\forall\sigma\in S_n\). 
```

Comment:
_Replace lines 22-23 with:_ Protects against errors in the Kendall tau distance on the space of permutations. The Kendall distance between permutations (\sigma) and (\pi) is defined as the minimum number of adjacent transpositions required to change (\sigma) into (\pi).